src/reistijden_v1/models.py

```diff
@@ -0,0 +1,77 @@
+from django.db import models
+
+
+class Publication(models.Model):
+    publication_type = models.CharField(max_length=255)  # one of travelTime or trafficFlow
+    publication_reference_id = models.CharField(max_length=255)  # e.g. PUB_AMS_ACTUAL_TRAJECTORY_TT
+    publication_reference_version = models.CharField(max_length=255)  # e.g. "1.0"
+    publication_time = models.DateTimeField()
+    measurement_start_time = models.DateTimeField()
+    measurement_end_time = models.DateTimeField()
+
+
+class Measurement(models.Model):
+    publication = models.ForeignKey('Publication', on_delete=models.CASCADE)
+    measurement_site_reference_id = models.CharField(max_length=255)  # e.g. "SEC_0001"
+    measurement_site_reference_version = models.CharField(max_length=255)  # e.g. "1.0"
+    measurement_site_name = models.CharField(max_length=255, null=True)
+    measurement_site_type = models.CharField(max_length=255)  # e.g. "section"
+    length = models.IntegerField(null=True)
```

Comment:
Wat is de eenheid van length? Of is dat duidelijk zodra je ook maar iets van het project af weet?

Reply:
Ik heb geen idee. Is wel een goeie om erbij te zetten. Ik vraag het de opdrachtgever en zet het er later bij.